pkg/apis/core/validation/validation.go

```diff
@@ -5597,6 +5598,37 @@ func validatePodResourceClaimStatuses(statuses []core.PodResourceClaimStatus, po
 	return allErrs
 }
 
+// validatePodExtendedResourceClaimStatus validates the ExtendedResourceClaimStatus in a pod status.
+func validatePodExtendedResourceClaimStatus(status *core.PodExtendedResourceClaimStatus, containers []core.Container, fldPath *field.Path) field.ErrorList {
+	if status == nil {
+		return nil
+	}
+	var allErrs field.ErrorList
+	rmFldPath := fldPath.Child("requestMapping")
+	for i, rm := range status.RequestMapping {
```

Comment:
missing validation to ensure items in the list are unique by {containerName,extendedResourceName}, right?

Reply:
added validation to ensure unique by {containerName,extendedResourceName}